ci/steps/notify.py

```diff
@@ -0,0 +1,136 @@
+import ccc.github
+import ci.util
+import distutils.util
+from email import encoders
+from email.mime.base import MIMEBase
+import glci.notify
+import glci.util
+import json
+import mailutil
+import os
+from string import Template
+import typing
+import urllib
```

Comment:
nit: sorting (I think its done here as we do it in cc-utils so far)

Reply:
sorting is good but where is the order incorrect here?